app/dashboard/templates/yge/send2.html

```diff
@@ -55,11 +55,11 @@ <h1>Send Tip.</h1>
             </div>
             <div>
                 From Name (optional):
-                <input type="text" placeholder="Bobby Tables" id="fromName" value="" style="width: 400px; margin-left: auto; margin-right: auto;">
+                <input type="text" placeholder="Bobby Tables" id="fromName" value="{% if from_handle %}{{ from_handle }}{% endif %}" style="width: 400px; margin-left: auto; margin-right: auto;">

```

Comment:
strangely enough.. i cant get this information to prepopulate after i auth with github.. wonder what im doing wrong

Reply:
I'll take a look when I'm going the refactor and include setup steps/gifs of functionality.